File: benchmarks/vm/05_call_loop.rb
def mix(x)
  (x * 3) + 1
end

t0 = Process.clock_gettime(Process::CLOCK_MONOTONIC)
total = 0
100_000.times { |i| total += mix(i) }
total_ms = (Process.clock_gettime(Process::CLOCK_MONOTONIC) - t0) * 1000
puts "total = #{total}"
puts format("BENCH_RESULT: %.3f ms", total_ms)

File: benchmarks/vm/05_call_loop.lua
local function mix(x)
    return (x * 3) + 1
end

local t0 = os.clock()
local total = 0
for i = 0, 99999 do
    total = total + mix(i)
end
local total_ms = (os.clock() - t0) * 1000
print("total = " .. tostring(total))
print(string.format("BENCH_RESULT: %.3f ms", total_ms))
